src/clr-angular/layout/main-container/_layout.clarity.scss

```diff
@@ -79,11 +79,12 @@
     }
   }
 
-  body.no-scrolling .main-container .content-container .content-area {
+  body.no-scrolling,
+  body[cds-layout='no-scrolling'] {
     overflow: hidden;
-  }
 
-  body.no-scrolling {
-    overflow: hidden;
+    .main-container .content-container .content-area {
```

Comment:
We should note that this selector specifically targets Clarity-UI and is here only for integration between the two. Core will not be carrying this DOM structure over.

Reply:
Thank you. Yes, I removed that in the core's scss file. I'll put a comment here as well for extra clarity.